include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
No, it's still an error.  The only time it would work is if an integer constant of smaller size is used (eg. `1LL` will error, `1L` or `1` would work).  Using a variable or return value of `long long` type will still error regardless.

Additionally, this code is also used for C, so constructors are illegal.  If one was added it'd have to get protected in a further `#ifdef __cplusplus` block, which just complicates things further.